src/bp/ui-studio/src/web/components/Confusion/style.scss

```diff
@@ -0,0 +1,15 @@
+.yellow {
```

Comment:
you could use bootstrap or blueprint helper class for this instead of writing your own css + it ensures we keep the same colors all around